server/StudySharp.API/Requests/TheoryBlocks/RemoveTheoryBlockByIdRequest.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace StudySharp.API.Requests.TheoryBlocks
+{
+    public class RemoveTheoryBlockByIdRequest
+    {
+        [BindProperty(Name = "id", SupportsGet = true)]
```

Comment:
Have you checked the controller functionality with the help of Swagger? Does every method go well? Because here we have SupportsGet flag. Not sure if it will work with HttpDelete, so that's why I'm asking

Reply:
Not yet